### PR TITLE
Removed the concept of Dynamic Postgresql

### LINKED ISSUE #3497: [Docs RFC] Remove mentions of  Dynamic Postgres from the docs
Dynamic postgres is now plain old Postgres. 

Needs to go live on the 15th Oct.

### PR DESCRIPTION
**Pull request by atovpeko**
Removed the page about Dynamic PostgreSQL and links to it. 
UI mentions remain until the UI update.

### CODE REVIEW

File: _partials/_cloud-intro.md [hunk 1]
```diff
@@ -5,13 +5,11 @@ scale further, and stay under budget. Timescale Cloud offers the following Postg
   [time-series and analytics][what-is-time-series] workloads. Get automated dynamic data partitioning, hybrid 
   row-columnar storage, advanced compression techniques, incremental up-to-date materializations, and specialized 
   analysis functions as well as cloud-only features like transparent tiering and low-cost object storage.
-- **[Dynamic PostgreSQL][create-service]**: for all other workloads. A PostgreSQL instance with a
-  [dynamic compute range][what-is-dynamic-postgres] aligned to your business needs. You select a compute range, only 
+- **[Dynamic PostgreSQL][create-service]**: for all other workloads. A PostgreSQL instance with a dynamic compute range aligned to your business needs. You select a compute range, only
```
Comment (billy-the-fish) [inline _partials/_cloud-intro.md:8]: I thought the point was to remove mentions of Dynamic Postgres. Can you have a check through and update to do:
* Time series
* Vector and AI
* Postgres
And only leave a mention of dynamic for the get started where that is still in the UI.

File: use-timescale/services/index.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ import CloudIntro from "versionContent/_partials/_cloud-intro.mdx";
 
 Read about Timescale features in the documentation:
 
-*   Determine if you want a time-series or [Dynamic PostgreSQL][dynamic-pg] database
+*   Determine if you want a time-series or regular PostgreSQL database
```
Comment (billy-the-fish) [inline use-timescale/services/index.md]:
```suggestion
*   Determine if you want a time-series, ai or regular PostgreSQL database
```

File: _partials/_cloud-intro.md [hunk 3]
```diff
@@ -1,17 +1,12 @@
...
 scale further, and stay under budget. Timescale Cloud offers the following PostgreSQL optimized database services:
 
-- **[Time-series and Analytics][create-service]**: a Timescale DB instance optimized for your
-  [time-series and analytics][what-is-time-series] workloads. Get automated dynamic data partitioning, hybrid 
-  row-columnar storage, advanced compression techniques, incremental up-to-date materializations, and specialized 
-  analysis functions as well as cloud-only features like transparent tiering and low-cost object storage.
-- **[Dynamic PostgreSQL][create-service]**: for all other workloads. A PostgreSQL instance with a
-  [dynamic compute range][what-is-dynamic-postgres] aligned to your business needs. You select a compute range, only 
-  paying for the base and the amount of extra CPU as you scale.
+- **[Time-series and analytics][create-service]**: for storing and querying [time-series data][what-is-time-series] at scale. Get faster time-based queries with hypertables, continuous aggregates, and columnar storage. Save on storage with native compression, data retention policies, and bottomless data tiering to Amazon S3.
```
Comment (solugebefola) [inline _partials/_cloud-intro.md]: Given the adoption of the add-on model, I think we need to rethink how we present plain PostgreSQL here?  It implies that Time-series and AI _don't_ have the abilities of PostgreSQL?  I know it's stated right above that they all are "PostgreSQL optimized", so maybe the PostgreSQL bullet is called "Standard PostgreSQL".  Might want to run that by Marketing folks

File: _partials/_cloud-intro.md [hunk 4]
```diff
@@ -1,17 +1,12 @@
...
-  paying for the base and the amount of extra CPU as you scale.
+- **[Time-series and analytics][create-service]**: for storing and querying [time-series data][what-is-time-series] at scale. Get faster time-based queries with hypertables, continuous aggregates, and columnar storage. Save on storage with native compression, data retention policies, and bottomless data tiering to Amazon S3. 
+- **[AI and vector][create-service]**: for building AI applications from start to scale. Get fast and accurate similarity search with the pgvector and pgvectorscale extensions. Create vector embeddings and perform LLM reasoning on your data with the pgai extension.
+- **[PostgreSQL][create-service]**: for applications requiring strong data consistency, complex relationships, and advanced querying capabilities. A trusted industry-standard RDBMS with ACID compliance, extensive SQL support, JSON handling, and extensibility through custom functions, data types, and extensions.
 
 All databases are extended with lightning fast vector search, and include all the cloud tooling you'd expect for production use,
```
Comment (solugebefola) [inline _partials/_cloud-intro.md]: Does "lightning fast vector search" apply to all databases or only TimescaleDB databases?  Sorry that's a tangent

File: use-timescale/services/index.md [hunk 5]
```diff
@@ -18,7 +18,7 @@ import CloudIntro from "versionContent/_partials/_cloud-intro.mdx";
 
 Read about Timescale features in the documentation:
 
-*   Determine if you want a time-series or [Dynamic PostgreSQL][dynamic-pg] database
+*   Determine if you want a time-series, AI, or plain PostgreSQL database.
```
Comment (solugebefola) [inline use-timescale/services/index.md]:
```suggestion
*   Determine if you want a time-series, AI, or standard PostgreSQL database.
```
?  Again, may want to run it by Marketing folks

**Review by solugebefola — APPROVED** [review]
Just a reminder that this won't be the interface that most folks see until the experiment is concluded and if it's successful (signs point to yes, but)

File: _partials/_cloud-intro.md (hunk 3 above)
Comment (billy-the-fish) [reply _partials/_cloud-intro.md]: Point out that AI and Times-series are extensions on top of [PostgreSQL]? Makes sense. @jamessewell, could you share your wisdom please.

File: _partials/_cloud-intro.md (hunk 4 above)
Comment (billy-the-fish) [reply _partials/_cloud-intro.md]: @jamessewell, could you share your wisdom here as well please :-)

File: use-timescale/services/index.md (hunk 5 above)
Comment (billy-the-fish) [reply use-timescale/services/index.md]: @jamessewell, and again :-)

File: _partials/_cloud-intro.md (hunk 3 above)
Comment (jamessewell) [reply _partials/_cloud-intro.md]: I think this needs to match what people see on the create service page (I'd pull that up when we edit this)

So

PostgreSQL as the base
  - enable time-series
  - enable AI

File: _partials/_cloud-intro.md (hunk 4 above)
Comment (jamessewell) [reply _partials/_cloud-intro.md]: It used to apply to all, now it's only ones which have the AI option enabled.

File: getting-started/services.md [hunk 6]
```diff
@@ -15,12 +15,9 @@ import ServiceOverview from "versionContent/_partials/_service-overview.mdx";
 
 Timescale Cloud offers the following PostgreSQL optimized database services:
 
-- **Time-series and Analytics**: a Timescale DB instance optimized for your 
-    [time-series and analytics][what-is-time-series] workloads.
-- **Dynamic PostgreSQL**: for all other workloads. A PostgreSQL instance with a 
-    [dynamic compute range][what-is-dynamic-postgres] aligned to your business needs.
-- **Vector and AI**: for apps that require fast search on vector embeddings and metadata, like semantic search, image
-  search, RAG and Agents. Powered by Timescale Vector and available on Time-series and Dynamic PostgreSQL services. 
+- **Time-series and analytics**: for storing and querying [time-series data][what-is-time-series] at scale. Get faster time-based queries with hypertables, continuous aggregates, and columnar storage. Save on storage with native compression, data retention policies, and bottomless data tiering to Amazon S3.
```
Comment (jamessewell) [inline getting-started/services.md]: Same comment as above, Postgres is the base, time-series and ai are add-ons.

I've screen-shotted the three configs below so you can see the product wording.

Screenshot 2024-11-19 at 8 41 00 pm
Screenshot 2024-11-19 at 8 41 37 pm

Screenshot 2024-11-19 at 8 41 48 pm

File: use-timescale/services/index.md (hunk 5 above)
Comment (jamessewell) [reply use-timescale/services/index.md]: Maybe something like

* Decide if you want to add time-series or AI capabilities to your standard PostgreSQL database?

**Comment by jamessewell** [comment]
🚨 Ok my comments above were for the new flow. @solugebefola that's only for 50% of folks though right?

This is really hard with two seperate flows which are very different

**Comment by billy-the-fish** [comment]
@jamessewell , @solugebefola , we know that the get started cannot be correct as the two flows are so different. We are addressing that change in https://github.com/timescale/docs/pull/3533/files, which will be published when the new flow is decided. This PR is mainly to remove https://github.com/timescale/docs/pull/3500/files#diff-2920e246768315870aec9bd05b085809ac7b87e1689345dcf0a5ceeca17f5c8b from the docs and make it as not-wrong as possible :-).  I believe that this PR is technically true as these features can be/are included in the services no matter which UI.

File: _partials/_cloud-intro.md (hunk 3 above)
Comment (billy-the-fish) [reply _partials/_cloud-intro.md]: I've updated this so service capabilities match the wording in the UI. These statements are true of the old UI so I think it passes.

File: use-timescale/services/index.md (hunk 2 above)
Comment (billy-the-fish) [reply use-timescale/services/index.md]: Removed. It makes no sense.

File: use-timescale/services/index.md (hunk 5 above)
Comment (billy-the-fish) [reply use-timescale/services/index.md]: Removed. It makes no sense in the context.

File: _partials/_cloud-intro.md (hunk 4 above)
Comment (billy-the-fish) [reply _partials/_cloud-intro.md]: Removed the mention of vector search and updated to link to cloud features.

File: getting-started/services.md (hunk 6 above)
Comment (billy-the-fish) [reply getting-started/services.md]: Replaced with an include to the _cloud-intro. Text is now the same.

**Review by billy-the-fish — APPROVED** [review]
LGTM

**Pull request merged**